src/distinct.rs

```diff
@@ -377,10 +377,10 @@ impl DistanceResult {
             (self.closest_distances.len() - self.num_fixed_colors) as Scalar;
     }
 
-    fn distance(&self, a: &(Color, Lab), b: &(Color, Lab)) -> Scalar {
+    fn distance(&self, a: &Lab, b: &Lab) -> Scalar {
         match self.distance_metric {
-            DistanceMetric::CIE76 => delta_e::cie76(&a.1, &b.1),
-            DistanceMetric::CIEDE2000 => delta_e::ciede2000(&a.1, &b.1),
+            DistanceMetric::CIE76 => delta_e::cie76(&a, &b),
```

Comment:
Could you please fix the clippy warning here: "this expression creates a reference which is immediately dereferenced by the compiler"